components/modal/Modal.tsx

```diff
@@ -57,7 +57,7 @@ export interface ModalFuncProps {
   iconType?: string;
 }
 export type ModalFunc = (props: ModalFuncProps) => {
-  destroy: () => void
+  destroy: (e) => void
```

Comment:
ref: https://github.com/ant-design/ant-design/issues/2110

Here is a problem, if developers call destroy, should he pass an event to it?